src/abe_policy/dimension.rs

```diff
@@ -173,6 +173,26 @@ impl Dimension {
         }
     }
 
+    pub fn restrict(&self, attr_name: AttributeName) -> Result<Self, Error> {
```

Comment:
Maybe add a comment to explain what this function is intended for

Reply:
done in 314c441